app/components/namespace_tree/row/row_contents_component.html.erb

```diff
@@ -0,0 +1,57 @@
+<% if @namespace.type == "Group" %>
+  <%= viral_icon(name: :squares_2x2, classes: "h-5 w-5 text-slate-400 mr-2") %>
+  <%= viral_avatar(
+    name: @namespace.name,
+    size: :small,
+    colour_string: "#{@namespace.name}-#{@namespace.id}",
+    data: {
+      turbo: false
+    },
+    url: group_path(@namespace)
+  ) %>
+<% else %>
+  <%= viral_icon(name: :rectangle_stack, classes: "h-5 w-5 text-slate-400 mr-2") %>
+  <%= viral_avatar(
+    name: @namespace.name,
+    size: :small,
+    colour_string: "#{@namespace.name}-#{@namespace.id}",
+    data: {
+      turbo: false
+    },
+    url: project_path(@namespace.project)
+  ) %>
+<% end %>
+
+<div class="flex items-center ml-2 namespace-text-container grow">
+  <div class="flex flex-col namespace-text">
+    <div class="namespace-text grow shrink">
+      <div class="flex flex-wrap items-center mr-3 font-semibold title">
+        <%= link_to @namespace.name, group_path(@namespace) %>
+      </div>
+      <div class="description">
+        <p><%= @namespace.description %></p>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="flex flex-wrap justify-between flex-none metadata shrink-0">
+  <div class="grid grid-cols-2 gap-3 mt-2 stats text-slate-500 group-stats">
+    <% if @namespace.type == "Group" %>
+      <%= viral_tooltip(title: t(:'.stats.projects')) do %>
+        <span class="flex items-center text-sm">
+          <%= viral_icon(name: :rectangle_stack, color: :subdued, classes: "h-5 w-5 mr-1") %>
+          <%= Namespace.where(
```

Comment:
**nitpick (blocking):** Can you change this to `@namespace.project_namespaces.count`.

Reply:
Updated in e94d13c